core/controllers/classifier.py

```diff
@@ -79,7 +79,10 @@ def verify_signature(message, vm_id, received_signature):
     if secret is None:
         return False
 
-    generated_signature = generate_signature(secret, message)
+    if message is None:
```

Comment:
Ah, I see. I didn't try it actually. I just assumed that since there is not message what are we enciphering here but since it seems working let's go with that. Empty `message` in request. Thanks for confirming this @seanlip